css/styles.css

```
@@ -0,0 +1,52 @@
+*{
+    margin: 10px;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: rgb(254, 254, 254);
+}
+
+.error-message {
+    color: red;
+    font-weight: bold;
+    margin-bottom: 10px;
+}
+
+
+.task-form {
+    display: none;
+    flex-direction: column;
+}
+
+.task-form.show {
+    display: flex;
+}
+
+.task-card {
+    border: 1px solid #ddd;
+    padding: 10px;
+    margin: 10px 0;
+    position: relative;
```

```suggestion
    border: 1px solid #ddd;
    margin: 10px 0;
    padding: 10px;
    position: relative;
```

Sort CSS properties alphabetically.